addons/sale/views/sale_portal_templates.xml

```diff
@@ -595,4 +596,10 @@
             <t t-call="account.document_tax_totals"/>
         </table>
     </template>
+
+    <template id="sale_order_portal_content_loyalty_table">
+        <table class="table table-sm">
+            <t t-call="sale_loyalty.points_table"/>
```

Comment:
This template does not exist without `sale_loyalty` it can't be used directly in `sale`